.eslintrc

```diff
@@ -72,6 +72,12 @@
     "jsx-a11y/no-static-element-interactions": "off",
     "no-console": "error",
     "no-plusplus": "off",
+    "no-restricted-syntax": [
+      'error',
+      'ForInStatement',
```

Comment:
Oh, huh, thanks for linking to the comment. I wish there was a way to make this more obvious to the next person who accidentally removes them all but I got nothing. I put the ignore comments back.